### PR TITLE
Allow --ca-cert arg for updating AMI/image

### PR DESCRIPTION
**Pull request by bonzotronix**

### CODE REVIEW

File: brkt_cli/instance_config_args.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def instance_config_from_values(values=None, mode=INSTANCE_CREATOR_MODE,
         ic.add_brkt_file('proxy.yaml', proxy_config)
 
     if 'ca_cert' in values and values.ca_cert:
-        if mode != INSTANCE_CREATOR_MODE:
+        if mode not in (INSTANCE_CREATOR_MODE, INSTANCE_UPDATER_MODE):
```
Comment (bburtin) [inline brkt_cli/instance_config_args.py]: Do we even need this check, since we're selectively adding the option?

Comment (bonzotronix) [inline brkt_cli/instance_config_args.py]: I could remove it.  I guess having that check reminds us of the restriction.  But I'm OK either way.  Remove?

Comment (bburtin) [inline brkt_cli/instance_config_args.py]: Yeah, I generally think that less code is a good thing.  Thanks!

**Comment by bonzotronix** [comment]
OK, PTAL

**Comment by bburtin** [comment]
👍

**Pull request merged**